saturnbase-gpu/install-miniconda.bash

```diff
@@ -18,10 +18,9 @@ if ! echo "${MD5SUM}  ${INSTALLER_PATH}" | md5sum  --quiet -c -; then
 fi
 
 bash ${INSTALLER_PATH} -b -p ${CONDA_DIR}
-export PATH="${CONDA_DIR}/bin:$PATH"
 
```

Comment:
I don't understand why this is ok to do in install-jupyter.bash and not ok in install-miniconda.bash

Reply:
Fine to do in either really. I pulled it out of `install-miniconda.bash` because I wanted to make everything consistent, but then the build failed because some part of the install wasn't able to find npm (since it wasn't on the PATH yet). Adding it there just puts the conda bin on the PATH for the script without affecting the image build.

Can revert the change to `install-miniconda.bash` if preferred.

Reply:
@jsignell it is the other way around actually...it's not forbidden but just not needed to set PATH here (wont hurt if you prefer tho), which is needed for install-jupyter as Ben explained.

Reply:
I'd prefer to revert this then.

Reply:
done